### PR TITLE
PMM-5171: Remote instance addition required fields update

### PR DESCRIPTION
**Pull request by lunaticusgreen**

### CODE REVIEW

**Comment by puneet0191** [comment]
@lunaticusgreen where is FB for this?

**Comment by lunaticusgreen** [comment]
> Please add link to FB created for these changes?

Of course:
https://github.com/Percona-Lab/pmm-submodules/pull/584

File: pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx [hunk 1]
```diff
@@ -186,10 +186,10 @@ const AddRemoteInstance = props => {
               <span className="description">Port your service is listening on</span>
             </div>
             <div className="add-instance-panel">
-              <InputField form={form} name="username" placeholder="*Username" required={true} />
+              <InputField form={form} name="username" placeholder="Username" required={true} />
               <span className="description">Your database user name</span>
 
-              <PasswordField form={form} name="password" placeholder="*Password" required={true} />
+              <PasswordField form={form} name="password" placeholder="Password" required={true} />
```
Comment (puneet0191) [inline pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx]: Username and Password fields are required as far as I know, why are we removing the * asterisk? or I am getting it wrong?

Comment (lunaticusgreen) [reply pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx]: I just changed functionality a little, now Input field component adds atersik if required flag is used

**Pull request merged**